fdbserver/workloads/LocalRatekeeper.actor.cpp

```diff
@@ -0,0 +1,119 @@
+#include "workloads.actor.h"
+#include <fdbserver/Knobs.h>
+#include <flow/actorcompiler.h>
+
+namespace {
+
+ACTOR Future<StorageServerInterface> getRandomStrage(Database cx) {
```

Comment:
`getRandomStrage` should be `getRandomStorage`

Reply:
Fixed, thanks